smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -269,21 +269,24 @@ contract IPublicLock is IERC721Enumerable {
     uint amount
   ) external;
 
-  /**
-   * @notice Destroys the msg.sender's key and sends a refund based on the amount of time remaining.
+   /**
+   * @dev Destroys the key managed by msg.sender and sends a refund
+   * based on the amount of time remaining.
    */
-  function cancelAndRefund() external;
+  function cancelAndRefund(uint) external;
 
   /**
-   * @dev Cancels a key owned by a different user and sends the funds to the msg.sender.
-   * @param _keyOwner this user's key will be canceled
-   * @param _v _r _s getCancelAndRefundApprovalHash signed by the _keyOwner
+   * @dev Cancels a key managed by a different user and sends the funds to the msg.sender.
```

Comment:
Pretty sure I wrote this originally... but the comment is wrong. the funds are sent to the keyOwner, not msg.sender.

Reply:
good catch. fixed.